app/controllers/user_api_keys_controller.rb

```diff
@@ -71,7 +71,7 @@ def create
       client_id: params[:client_id],
       user_id: current_user.id,
       push_url: params[:push_url],
-      scopes: scopes
+      scopes: scopes.map { |name| UserApiKeyScope.new(name: name) }
```

Comment:
Is that going to create an N+1?

Reply:
Yes it will create 1 SQL query for creating the key, and N queries for creating each scope. However, the number of scopes is limited to the number of scopes we actually have (currently 7), and since this only happens during key **creation** I think it is ok?

I have confirmed that scopes are being preloaded correctly when **reading** from the database, so there is no N+1 on GET requests.